CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+### 2.1.0 (unreleased)
+
+New features:
```

Comment:
I think a header would look nicer here. I think it'll also enable linking to a particular release (`/CHANGELOG.md#2.1.0`).

I'd consider the following formatting:

```
# Changelog

## 2.1.0 (unreleased)

### New features

- `WhitelistCrowdsale`
```

<details>
<summary>Rendered</summary>

# Changelog

## 2.1.0 (unreleased)

### New features

- `WhitelistCrowdsale`
</details>

Reply:
Good thinking!